front_end/html/edit_course.html

```diff
@@ -45,6 +45,6 @@ <h2>
       <a href="/delete_course/{{ course_basics["id"] }}">Delete course</a> |
       <a href="/delete_course_submissions/{{ course_basics["id"] }}">Delete course submissions</a>
     {% end %}
-</p>
+</p></br>
```

Comment:
There's actually no such thing as a closing br tag. But you can have a self-closing tag`<br />`. Perhaps that's what you meant. Please play with it a bit more to see what will work. Please also check on two different browsers to make sure it looks right on both.